<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Query</class>
 <widget class="QWidget" name="Query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <property name="windowIcon">
   <iconset resource="libres.qrc">
    <normaloff>:/lib/fdogicon.png</normaloff>:/lib/fdogicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:Microsoft YaHei;}
#Query{
background-color: rgb(193, 255, 183);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_2{
background-color: rgb(228, 241, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,10">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(120, 192, 255);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="libres.qrc">
               <normaloff>:/lib/fdogicon.png</normaloff>:/lib/fdogicon.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>12</pointsize>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>查找</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_2">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="libres.qrc">
               <normaloff>:/lib/zuixiaohua.png</normaloff>:/lib/zuixiaohua.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_4">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="libres.qrc">
               <normaloff>:/lib/guanbi.png</normaloff>:/lib/guanbi.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>151</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_3{
border-style:solid;
border-width:2px;
	border-bottom-color: rgb(198, 198, 198);
}
</string>
        </property>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>70</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(110, 149, 255);
border-style:solid;
border-width:1px;
border-radius:5px;</string>
         </property>
         <item>
          <property name="text">
           <string>不限</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12-16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17-21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22-30</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>500</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(110, 149, 255);
border-style:solid;
border-width:1px;
border-radius:5px;</string>
         </property>
         <property name="placeholderText">
          <string>输入Fdog查找</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>70</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(110, 149, 255);
border-style:solid;
border-width:1px;
border-radius:5px;</string>
         </property>
         <item>
          <property name="text">
           <string>不限</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>男</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>女</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>70</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(110, 149, 255);
border-style:solid;
border-width:1px;
border-radius:5px;</string>
         </property>
         <property name="placeholderText">
          <string>故乡：江苏</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>70</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(110, 149, 255);
border-style:solid;
border-width:1px;
border-radius:5px;</string>
         </property>
         <property name="placeholderText">
          <string>中国-江苏-泰州</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>50</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(111, 188, 255);
border-style:solid;
color: rgb(255, 255, 255);
border-radius:5px;</string>
         </property>
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>20</y>
           <width>57</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>在线</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>60</y>
           <width>73</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(130, 178, 255);
border-style:solid;
color: rgb(255, 255, 255);
border-radius:5px;
</string>
         </property>
         <property name="text">
          <string>添加好友</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>66</width>
           <height>66</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:33px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>66</width>
           <height>66</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="libres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Query</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
